### PR TITLE
Update manage-backup-restore.adoc

### PR DESCRIPTION
**Pull request by aqurishi-mulesoft**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 1]
```diff
@@ -62,12 +62,19 @@ scp your_username@remotehost:/opt/anypoint/runtimefabric/backup.tar.gz /backup-p
...
 
+[WARNING]
+====
+Restore must be performed to the same Runtime Fabric Cluster in Control Plane (Same Runtime Fabric cluster in Runtime Manager)
+====
+
 Runtime Fabric provides two target options when restoring a cluster from a backup:
 
-* Use an existing Runtime Fabric cluster.
+* Use an existing Runtime Fabric cluster [underlying infrastructure].
```
Comment (IsaacEldridge) [inline modules/ROOT/pages/manage-backup-restore.adoc]: @aqurishi-mulesoft I'm not sure what this additions means. Can you clarify for me?

**Review by IsaacEldridge — COMMENTED** [review]
@aqurishi-mulesoft please take a look at my edits

File: modules/ROOT/pages/manage-backup-restore.adoc (hunk 1 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/manage-backup-restore.adoc]: ok to remove

**Review by IsaacEldridge — DISMISSED** [review]

File: modules/ROOT/pages/manage-backup-restore.adoc [hunk 2]
```diff
@@ -62,12 +62,19 @@ scp your_username@remotehost:/opt/anypoint/runtimefabric/backup.tar.gz /backup-p
 
 == Restore a Cluster
 
+[WARNING]
+====
+Restore must be performed to the same Runtime Fabric Cluster in Control Plane (Same Runtime Fabric cluster in Runtime Manager)
```
Comment (bjoson-mulesoft) [inline modules/ROOT/pages/manage-backup-restore.adoc]: This sentence confuses me a little. Is it:
You must restore the Runtime Fabric Cluster to the same control plane or Runtime Manager in which it originated.

**Review by bjoson-mulesoft — DISMISSED** [review]

**Review by bjoson-mulesoft — APPROVED** [review]

**Pull request merged**